src/components/Popups/SurveyPopup.tsx

```diff
@@ -51,7 +51,7 @@ const WrappedCloseIcon = styled(X)`
 
 const END_TIMESTAMP = 1642272346 // Jan 15th
 
-export default function SurveyPopup() {
+function SurveyPopup() {
   const theme = useTheme()
```

Comment:
How did that pass your local linter 🫣

Reply:
This is really interesting, we now have a file that has no exported members... I would assume it should be automatically removed!

Reply:
it's because the other thing is just a warning, not an error!